code/style.css

```diff
@@ -2,6 +2,466 @@
+}
+
+li {
+  list-style-type: dot;
+  font-size: 18px;
+  line-height: 1.5;
+  padding: 0px;
+  margin: 5px;
+}
+
+/* ACCORDION */
+.accordion-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center; 
+  background-color: #F0E3C7;
+  padding-bottom: 40px;
+}
+
+.question {
+  text-align: left;
+  cursor: pointer;
+}
+
+.answer-container {
+  background-color: white;
+  display: none;
+  text-align: center;
+  padding-left: 5px;
+  width: 40%;
+  border-radius: 25px;
+}
+
+.answer {
+  font-size: 16px;
+}
+
+.active + .answer-container {
+  display: block;
+  width: 40%;
+  padding: 20px;
+}
+
+/* FORM */
+.form-container {
+  display: flex;
+  flex-wrap: wrap;
+  background-color: #F0E3C7;
+  padding: 0px 0px 60px 100px;
+  }
+
+.form1 {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  padding-left: 100px;
+  width: 50%;
+  }
+
+.form2 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 80px 0px 0px 100px;
+  width: 50%;
+}
+
+.form-heading {
+  font-size: 30px;
+  padding-top: 60px;
+  margin: 0px;
+}
+
+.form-intro {
+  font-size: 20px;
+  width: 75%;
+}
+
+.intro-link {
+  text-decoration: underline;
+}
+
+.intro-link:hover {
+  background-color: #cde7f7;
+  text-decoration: none;
+}
+
+.form-group {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+label {
+  display: flex;
+  font-weight: bold;
+  font-size: 16px;
+  padding: 20px 0px 10px 0px;
+  width: 75%;
+}
+
+input {
+  border: none;
+  cursor: pointer;
+  border-radius: 25px;
+  height: 40px;
+  width: 250px;
+  margin-bottom: 20px;
+}
+
+input:hover, #meddelande:hover {
+  background-color: #cde7f7;
+}
+
+#meddelande {
+  border: none;
+  margin-bottom: 20px;
+}
+
+button {
+  border: none;
+  font-weight: bold;
+  background-color: #cde7f7;
+  cursor: pointer;  
+  border-radius: 25px;
+  font-size: 20px;
+  height: 70px;
+  width: 180px;
+  margin-top: 10px;
+}
+
+button:hover {
+  background-color:white;
+}
+
+/* FOOTER */
+footer {
+  width: 100%;
+  background-color: #cde7f7;
+  padding: 60px 60px 100px 60px;
+}
+
+span {
+  display: block;
+  padding-top: 5px;
+}
+
+/* For tablet */
+@media (max-width: 1024px) {
+
+  .hero {
+  height: 250px;
+  }
+
+/* Content */
+  .card-container {
+  padding: 0px;
+  }
+
+  .card-right, .card-left {
+  margin: 30px;
+  }
+
+  .card-text {
+    font-size: 14px;
+  }
+
+  li {
+    font-size: 12px;
+  }
+
+  .card-link {
+    font-size: 14px;
+    padding: 15px 0px 15px 0px;
+    margin: 0px;
+  }
+
+  /* FORM */
+  .form-container {
+  align-items: center;
+  padding: 40px;
+  width: 100%;
+  }
+
+  .form-container1 .form-container2 {
+  align-items: center;
+  }
+
+  #namn, #mejl {
+  flex-direction: flex-start;
+  }
+
+  #godkännande {
+  margin: 0px;
+  flex-direction: flex-start;
+  }
+}
+
+/* For mobile */
+@media (max-width: 768px) {
+
+  h1.header-heading {
+    font-size: 48px;
+    width: 75%;
+  }
```

Comment:
Looks like you need an even smaller font-size:

<img width="366" alt="Skärmavbild 2020-09-08 kl  17 04 20" src="https://user-images.githubusercontent.com/51529201/92493893-57c48580-f1f5-11ea-9b76-e65f224e92bb.png">